### ISSUE TITLE
Cannot set template for collection added from Prismic source

### ISSUE THREAD

**Issue by juliageek** [gridsome/gridsome#1225]
## Description

I am using the `gridsome-source-graphql-prismic` source to import my Prismic schema into a Gridsome project.

### Steps to reproduce

I have the following configuration in my `gridsome.config.js`:

```
module.exports = {
  siteName: 'my-webiste-fe',
  plugins: [
    {
      use: 'gridsome-source-graphql-prismic',
      options: {
        url: process.env.PRISMIC_API_URL,
        fieldName: 'prismicio',
        typeName: 'prismicio',
        headers: {
          'Prismic-Ref': process.env.PRISMIC_REF, // useMasterRef will overload this line
          'Authorization': `${process.env.AUTH_TOKEN}`,
        },

        useMasterRef: true // undefined by default
      }
    }
  ],
templates: {
    prismicioProduct: [
      {
        path: '/products/:title',
        component: './src/templates/ProductInfo.vue'
      }
    ]
  }
}
```

### Expected result

I expect that the project would build and run with `npm run develop`. From reading the documentation, I understand that I need to prefix my collections with `prismicio` to indicate that they are part of the Prismic schema.

### Actual result

I get an error in the terminal that `Error: A content type for the prismicioProduct template does not exist.`

### Environment

System:
    OS: Linux 5.3 Ubuntu 18.04.4 LTS (Bionic Beaver)
    CPU: (8) x64 Intel(R) Core(TM) i5-8350U CPU @ 1.70GHz
  Binaries:
    Node: 12.16.3 - ~/.nvm/versions/node/v12.16.3/bin/node
    Yarn: 1.21.1 - /usr/bin/yarn
    npm: 6.14.4 - ~/.nvm/versions/node/v12.16.3/bin/npm
  Browsers:
    Chrome: 81.0.4044.129
    Firefox: 77.0.1
  npmPackages:
    @gridsome/source-graphql: ^0.1.0 => 0.1.0 
    gridsome: ^0.7.0 => 0.7.16 
    gridsome-source-graphql-prismic: ^0.1.6 => 0.1.6 
  npmGlobalPackages:
    @gridsome/cli: 0.3.3

**Comment by thetre97** [comment]
@juliageek I believe that that plugin is basically a copy of the `source-graphql` plugin, and it just merges a remote schema into Gridsome - so that data in the remote schema is never actually added to the Gridsome store, and therefore there won't be any type names available.
So you will need to manually create the pages using the `createPages` API: https://gridsome.org/docs/pages-api/#create-pages-from-graphql

**Comment by juliageek** [comment]
Actually, because this source doesn't import collections at all, I ended up writing my own source. Even for my own source, I had to create a path for each node in the collection in order to make the templates work. So, my final code for adding templates in `gridsome.config.js` is (my collection name is **Product** and my collection prefix is **Prismic**)
```
templates: {
    PrismicProduct: [
      {
        path: (node) => {
          return `/products/${node.slug}/info`
        },
        component: './src/templates/ProductInfo.vue',
      },
    ]
  }
```

**Comment by juliageek** [comment]
So this being said, I am not sure if this issue should still be considered a bug. Maybe it should be moved under "Improving documentation", as I think the documentation doesn't make it crystal clear that the collections are not imported in the schema and that, even if you get the collections into the schema, you still need to add manually the dynamic path to the node.

**Comment by thetre97** [comment]
Yes, I agree - it is just how it works, so not exactly a bug. But it could definitely do with more documentation on the plugins, and/or a section on the docs about it.

As for the paths, you will always need to set the template path but you should be able to do what you are doing above with the string definition instead (note the `:slug`):

```js
// https://gridsome.org/docs/templates/#setup-templates
module.exports = {
  templates: {
    PrismicProduct: [
      {
        name: 'info',
        path: '/product/:slug/info',
        component: './src/templates/ProductInfo.vue'
      }
    ]
  }
}
```
